### PR TITLE
Release with taskboot

### PR DESCRIPTION
**Pull request by La0**

### CODE REVIEW

File: .taskcluster.yml [hunk 1]
```diff
@@ -452,3 +452,40 @@ tasks:
...
+        $let:
+          version: {$eval: 'head_branch[10:]'}
+        in:
+          taskId: {$eval: as_slugid("release")}
+          dependencies:
+            - {$eval: as_slugid("addon_build")}
+            - {$eval: as_slugid("bot_build")}
+            - {$eval: as_slugid("backend_build")}
+            - {$eval: as_slugid("frontend_build")}
+            - {$eval: as_slugid("report_check_tests")}
```
Comment (marco-c) [inline .taskcluster.yml]: Nit: We could use the deploy tasks as dependencies, so we don't release unless everything is fully green.

Comment (La0) [reply .taskcluster.yml]: That won't work as the events are not the same.
The workflow should become: 
1. push a tag to build a release (and support pre-release as RC to push tests)
2. push the tag on testing/production & use the previously built images

The advantage is that you build less, and use "human" tags in TC hooks. But that's quite some work

Comment (marco-c) [reply .taskcluster.yml]: Ah right, here the situation is a bit different than the other repos. We deploy on push to testing/production, we release on tag. Forget about this then.

**Comment by La0** [comment]
It worked ! https://github.com/mozilla/code-coverage/releases/tag/test-1

**Pull request merged**